packages/core/admin/ee/server/bootstrap.js

```diff
@@ -24,12 +25,18 @@ const SSO_ACTIONS = [
   },
 ];
 
-module.exports = async () => {
+module.exports = async ({ strapi }) => {
   const { actionProvider } = getService('permission');
 
   if (features.isEnabled('sso')) {
     await actionProvider.registerMany(SSO_ACTIONS);
   }
 
+  if (features.isEnabled('audit-logs')) {
```

Comment:
Adding the service should happen in the register rather than the bootstrap don't you think ?